server/utils.py

```diff
@@ -319,6 +324,10 @@ def get_setting(name, default=None):
         str: Anything else.
 
     """
+
+    # Make sure migrations have run, otherwise return default
+    if not db_table_exists('server_sal_setting'):
```

Comment:
Actually this is the real problem the table name is `server_salsetting`. Swapping that over is resolving the endless loop error.